content/docs/catalogue/items.md

```diff
@@ -75,25 +75,143 @@ There are two tabs in the bottom part of the screen:
 - **Doses**: For vaccines, the number of dose per vial
 - **Vaccine** (check box): If this is checked, this item is a vaccine and a number of dose can be assigned
 
-##### Categories
+#### Categories
 
 - **ATC Category**: ATC stands for Anatomical, Therapeutic, Chemical and is a method of classifying entities, and identifying them by category
 - **Universal Name**: if the item is linked to the [universal drug code database](https://codes.msupply.foundation/), this shows the universal, or generic name for the given item
 - **Universal Code**: similarly, the code allocated to the universal drug code item
 
-##### Packaging
+#### Packaging
 
 - **Default pack size**: This is the default pack size that will be assigned to incoming stock as it is received
 - **Outer pack size**: The number of units in a carton (outer pack). Not the number of preferred pack size packs in a carton (outer pack).
 - **Volume per pack**: The default volume per preferred pack size pack. This is the volume that will be used by default when receiving goods. We recommend you divide a carton volume by the number of preferred pack size packs in a carton to get this figure. Open mSupply always stores volumes in m3 (cubic metres), but you can enter a volume as millilitres (ml) or litres (l) by entering the appropriate abbreviation after the number representing the volume. e.g. enter “0.5l” to enter a volume of 0.5 litres (= 0.0005m3).
 - **Volume per outer pack**: The default volume of an Outer pack size pack of this item
 - **Weight**: The weight of a preferred pack size pack in kg
 
-##### Pricing
+#### Pricing
 
 - **Margin**: This is the default margin that will be applied to this item on Inbound Shipments to calculate the selling price
 
-#### Master Lists
+### Master Lists
 
 This tab lists the Master Lists associated with this item.
 ![Item detail headers](/docs/catalogue/images/cat_itemmasterlists.png)
+
+## Item Variants
+
+On the [Open mSupply Central Server](/docs/getting_started/central), a third tab, Item Variants, is available. This tab allows you to configure variations of the item, such as different manufacturers or packaging sizes.
+
+Currently, Item Variants are only used for GAPS functionality.
```

Comment:
I guess this will be a link when there's an overview section?

Reply:
Yes thats the plan! 

Thanks for review James - yep will merge this as it is now to get some inspo for Adam for the GAPS overview page. I'm hoping we can squeeze in a PR to actually hide the item variants input for most users (if no variants are configured) and then maybe only mention the extra column in the GAPS specific docs?